requirements.txt

```diff
@@ -2,3 +2,4 @@
 setuptools==42.0.2
 zc.buildout==2.13.3
 wheel
+zpretty
```

Comment:
@tiberiuichim it should be sufficient to add the zpretty dependency in buildout only. The other three dependencies are necessary for the bootstrap.